usecases/config/config_handler.go

```diff
@@ -103,6 +103,7 @@ type Config struct {
 	DisableGraphQL                      bool                     `json:"disable_graphql" yaml:"disable_graphql"`
 	AvoidMmap                           bool                     `json:"avoid_mmap" yaml:"avoid_mmap"`
 	CORS                                CORS                     `json:"cors" yaml:"cors"`
+	DisableTelemetry                    bool                     `json:"telemetry" yaml:"telemetry"`
```

Comment:
are we gonna keep the yaml and json tags `telemetry` ?

Reply:
Good eye, fixed